experiments/process_sandbox/sandbox.hh

```diff
@@ -46,6 +46,51 @@ namespace sandbox
   struct SharedMemoryRegion;
   struct SharedPagemapAdaptor;
   struct MemoryProviderBumpPointerState;
+
+  /**
+   * A PAL that cannot be used to allocate memory.  This is used for sandboxes,
```

Comment:
It's hard to find what `PAL` means and this code is distant enough from snmalloc that it could warrant the expansion of the TLA.